services/bs-libs/utils/getLogger.ts

```diff
@@ -1,16 +1,32 @@
 import { createLogger, format, transports, Logger } from "winston";
+import * as chalk from "chalk";
 
-type LoggerService = "ClaimSubscriber" | "ClaimPublisher" | "SlackAlert";
+type LoggerService =
+	| "ClaimSubscriber"
+	| "ClaimPublisher"
+	| "SlackAlert"
+	| "EthDepositPoller";
 const instances = {} as Record<LoggerService, Logger>;
 
+const cennzBlue = chalk.hex("#1130FF");
+const cennzGreen = chalk.hex("#05b210");
+const cennzPurple = chalk.hex("#9847FF");
+
+const labels = {
+	ClaimSubscriber: cennzBlue("ClaimSubscriber"),
+	ClaimPublisher: cennzGreen("ClaimPublisher"),
+	SlackAlert: chalk.cyan("SlackAlert"),
+	EthDepositPoller: cennzPurple("EthDepositPoller"),
+};
```

Comment:
ooooh fancy, make sure the messages stored in `logs` folder has these colors stripped out